raiden-dapp/src/store/notifications/notification-context.ts

```diff
@@ -3,4 +3,5 @@ export enum NotificationContext {
   INFO = 'info',
   WARNING = 'warning',
   ERROR = 'error',
+  SETTLE = 'notification_settle',
```

Comment:
It's weird to see `SETTLE` as a context/level besides `INFO`, `WARNING`, etc... Here, it seems to be needed only to change the icon. May I suggest to change slightly the logic here, add an optional `icon` property to the Notification payload, and have the levels as the standardized keywords, with respective color codes (e.g. blue for info, yellow for warning, red for error, etc), and use the default icons only if the payload one is missing? This way, we won't be required to implement a new 'context' every time we want a custom icon on a notification, and can just commit the action with the property in the payload.